applications/FluidDynamicsApplication/custom_elements/qsvms_dem_coupled.cpp

```diff
@@ -0,0 +1,432 @@
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Joaquin Gonzalez-Usua
+//
+
+// Project includes
+#include "qsvms_dem_coupled.h"
+#include "includes/cfd_variables.h"
+#include "includes/dem_variables.h"
+#include "includes/checks.h"
+
+#include "custom_utilities/qsvms_dem_coupled_data.h"
+#include "custom_utilities/fluid_element_utilities.h"
```

Comment:
```suggestion
// System includes

// External includes

// Project includes
#include "qsvms_dem_coupled.h"
#include "includes/cfd_variables.h"
#include "includes/dem_variables.h"
#include "includes/checks.h"

// Application includes
#include "qs_vms_dem_coupled.h"
#include "custom_utilities/qs_vms_dem_coupled_data.h"
#include "custom_utilities/fluid_element_utilities.h"
```
(note the files renaming)